### PR TITLE
fixes #59 : improve menu bar

### LINKED ISSUE #59: improve menu bar
No specific figma so just copy from somewhere to make it prettier

### PR DESCRIPTION
**Pull request by keraliss**
Made some changes, not entirely sure about the colors, open to suggestion if anyone have any! 
Screenshot from 2023-12-05 16-09-23
Screenshot from 2023-12-05 16-09-34

### CODE REVIEW

**Comment by emjshrx** [comment]
Can we add social media next to orange bar and align the descriptions so that they appear aligned as a separate column?
Screenshot 2023-12-05 at 10 10 25 PM

**Comment by emjshrx** [comment]
nit: 
1. hover effects for social media icons 
2. different font color between page name and description

**Comment by keraliss** [comment]
A bit stumped about color. any suggestion? also i feel like black bg looks a bit bland. any other color combo for bg as well as texts?

**Comment by emjshrx** [comment]
> A bit stumped about color. any suggestion? also i feel like black bg looks a bit bland. any other color combo for bg as well as texts?

We have to use the color scheme, so we have only 4 possible choices. Peach maybe good

**Comment by keraliss** [comment]
Screenshot from 2023-12-06 17-34-19
thoughts?

**Comment by keraliss** [comment]
With some tweaking, this is looking much better imo

Screenshot from 2023-12-06 20-12-23

File: src/components/common/hamburger.tsx [hunk 1]
```diff
@@ -73,19 +91,50 @@ const SideMenu = () => {
...
       </a>
-      <div>
+      <div className="flex">
         <hr className="my-10 w-1/3 border-0 bg-orange outline outline-orange" />
+        <div className="my-6 hidden gap-4 px-10 lg:flex">
+          <a
+            href="https://twitter.com/bitshala_org"
+            target="_blank"
+          >
+            <FaTwitter className="text-4xl text-white hover:text-[#1DA1F2]" />
```
Comment (emjshrx) [inline src/components/common/hamburger.tsx:101]: Oh the hover colors are the companies colors, okay got it sorry

File: src/components/common/hamburger.tsx [hunk 2]
```diff
@@ -73,19 +91,50 @@ const SideMenu = () => {
...
-              className="text-2xl text-white hover:text-peach"
-            >
-              {link.name}
+            <a href={link.url} target={link.tar}>
+              <div className="flex ">
+                <p className="flex w-80 items-center font-header text-3xl font-bold text-white hover:text-orange lg:text-4xl">
+                  {link.name}
+                </p>
+                <p className="  mx-10 mt-1 hidden items-center justify-center text-xl text-peach  lg:flex">
+                  {" "}
```
Comment (emjshrx) [inline src/components/common/hamburger.tsx:134]: the `{" "}` acn be removed, one below as well

File: src/components/common/hamburger.tsx [hunk 3]
```diff
@@ -73,19 +91,50 @@ const SideMenu = () => {
...
+            href="https://twitter.com/bitshala_org"
+            target="_blank"
+          >
+            <FaTwitter className="text-4xl text-white hover:text-[#1DA1F2]" />
+          </a>
+          <a
+            href="https://github.com/bitshala"
+            target="_blank"
+          >
+            <FaGithub className="text-4xl text-white hover:text-[#6e5494]" />
```
Comment (emjshrx) [inline src/components/common/hamburger.tsx:107]:
```suggestion
            <FaGithub className="text-4xl text-white hover:rounded-full hover:bg-[black] hover:invert" />
```

**Review by emjshrx — APPROVED** [review]
ack

**Pull request merged**